src/domain/hub/HubContext/HubContext.tsx

```diff
@@ -50,7 +54,7 @@ interface HubProviderProps {}
 
 const NO_PRIVILEGES = [];
 
-const HubProvider: FC<HubProviderProps> = ({ children }) => {
+const HubContextProvider: FC<HubProviderProps> = ({ children }) => {
```

Comment:
Isn't it hub AND context?

Reply:
or HubWithContextProvider - you are not providing the context of a hub and servicing it, you are providing a Hub WITH its context and servicing it. It's misleading.